apache2/apache2_config.c

```diff
@@ -4032,6 +4045,13 @@ const command_rec module_directives[] = {
         CMD_SCOPE_ANY,
         "Set waf instanceId"
     ),
+    AP_INIT_TAKE1 (
+        "SecWafLockPerm",
```

Comment:
I think SecWafLockOwner might be a better name.

Reply:
will change